device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/ModuleClientTest.java

```diff
@@ -135,6 +138,38 @@ public void constructorThrowsForHTTP() throws URISyntaxException, ModuleClientEx
         new ModuleClient(connectionString, IotHubClientProtocol.HTTPS);
     }
 
+    @Test
+    public void constructorWithModelIdSuccess(final @Mocked System mockedSystem) throws URISyntaxException, IOException, ModuleClientException {
+        //arrange
+        final IotHubClientProtocol protocol = IotHubClientProtocol.AMQPS;
```

Comment:
use mqtt here instead

Reply:
will do

Reply:
Rethinking about this, it is actually fine to leave it as it is. Here we just want to make sure model id is set correctly with this constructor no matter which protocol